src/main/java/com/deavensoft/paketomat/courier/CourierServiceImpl.java

```diff
@@ -22,38 +22,39 @@ public class CourierServiceImpl implements CourierService {
     private final CenterService centerService;
     private final CityService cityService;
     private final DispatcherService dispatcherService;
-    private List<Package> packagesToDispatch = new ArrayList<>();
-    private List<City> citiesToDispatch = new ArrayList<>();
```

Comment:
These can be local variables and passed as arguments.

Reply:
Done.

Reply:
?

Reply:
Deleted.